docs/monitoring.md

```diff
@@ -269,17 +310,23 @@ for a running application, at `http://localhost:4040/api/v1`.
     <td>Details for the storage status of a given RDD</td>
   </tr>
   <tr>
-    <td><code>/applications/[app-id]/logs</code></td>
-    <td>Download the event logs for all attempts of the given application as a zip file</td>
+    <td><code>/applications/[base-app-id]/logs</code></td>
+    <td>Download the event logs for all attempts of the given application as a zip file.
+    </td>
   </tr>
   <tr>
-    <td><code>/applications/[app-id]/[attempt-id]/logs</code></td>
-    <td>Download the event logs for the specified attempt of the given application as a zip file</td>
+    <td><code>/applications/[base-app-id]/[attempt-id]/logs</code></td>
+    <td>Download the event logs for a specific application as a zip file</td>
```

Comment:
Looking a the the implementation in `FsHistoryProvider`, it states " If no attempt is specified, or there is no attemptId for attempts, return all attempts"

That is: the zip file contains the logs of *every* attempt. I'll add that to the text.